projects/lexical_plugin/lib/lexical/plugin/supervisor.ex

```diff
@@ -9,6 +9,6 @@ defmodule Lexical.Plugin.Supervisor do
   end
 
   def async(plugin_module, subject) do
-    Task.Supervisor.async_nolink(__MODULE__, plugin_module, :handle, [subject])
+    Task.Supervisor.async_nolink(__MODULE__, plugin_module, :diagnose, [subject])
```

Comment:
if the plugin type is not diagnostics in the future, will it use diagnose too?

Reply:
No, different plugins will hav different function names.